cloudcompose/cluster/aws/cloudcontroller.py

```diff
@@ -1,41 +1,62 @@
 from os import environ
+from os.path import abspath, dirname, join, isfile
 import logging
 from cloudcompose.exceptions import CloudComposeException
 import boto3
 import botocore
 from time import sleep
+import time, datetime
 from retrying import retry
 
 class CloudController:
     def __init__(self, cloud_config):
         logging.basicConfig(level=logging.ERROR)
         self.logger = logging.getLogger(__name__)
         self.cloud_config = cloud_config
-        config_data = cloud_config.config_data('cluster')
-        self.aws = config_data["aws"]
+        config_data = cloud_config.config_data('cluster')[0]
```

Comment:
This [0] seems like a bug. Do you have the config that needed this to work? I think it may be malformed.